ui/pages/permissions-connect/flask/snap-update/snap-update.js

```diff
@@ -48,6 +48,15 @@ export default function SnapUpdate({
     [request.permissions],
   );
 
+  const approvedPermissions = request.approvedPermissions ?? {};
+  const revokedPermissions = request.unusedPermissions ?? {};
+  const newPermissions = request.newPermissions ?? {};
+  const hasPermissions =
+    Object.keys(approvedPermissions).length +
```

Comment:
please add parentheses around arithmetic to avoid ambiguity.
(yes I know that `+` has higher precedence over `>`, but it makes the intentions clearer)

```typescript
      (
      Object.keys(approvedPermissions).length +
      Object.keys(revokedPermissions).length +
      Object.keys(newPermissions).length
      ) > 0;
```

Reply:
Not possible due to linting